chia/consensus/block_body_validation.py

```diff
@@ -184,7 +194,9 @@ async def validate_block_body(
             if block.transactions_info.generator_refs_root != generator_refs_hash:
                 return Err.INVALID_TRANSACTIONS_GENERATOR_REFS_ROOT, None
             if len(block.transactions_generator_ref_list) > constants.MAX_GENERATOR_REF_LIST_SIZE:
-                return Err.PRE_SOFT_FORK_TOO_MANY_GENERATOR_REFS, None
+                return Err.TOO_MANY_GENERATOR_REFS, None
+            if any([index >= height for index in block.transactions_generator_ref_list]):
```

Comment:
this should probably also need to be conditioned on being passed the freeze point

Reply:
No need, because we already checked for transactions_generator MUST be None before freeze point. And it is not None here.